app/src/integrationTests/java/com/example/networking/RestServiceIntegrationTest.java

```diff
@@ -38,7 +38,7 @@ public void afterEveryTest() throws IOException {
     public void shot_request_test_network_layer() throws IOException {
         mockWebServerHelper.enqueueJsonResponseFromFileForMockWebServer(mockWebServer, "shots_response.json");
 
-        List<ShotResponse> shotResponseList = restService.getShots().toBlocking().value();
+        List<ShotResponse> shotResponseList = restService.getShots().toObservable().blockingSingle();
```

Comment:
blockingGet does not work on continuous stream. I was not sure, tbh, if the testing model involves continuous stream or just a small block with a clear terminate.